jupyter-covid19-restrict-download/start-notebook.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+set -e
```

Comment:
@urvi-occ where did you copy `start-notebook.sh`, `start-singleuser.sh` and `start.sh` from and why do we need them?

Reply:
As I referenced in the PR description, I am referencing the whole image on [jupyterslim](https://github.com/uc-cdis/jupyter-superslim/blob/master/jupyter-superslim/Dockerfile) as I had to update python version